commit f5eaed6870c81db6a44fcbe79c887f11b2eb1e8e
Author: Andrew F. Davis <afd@ti.com>
Date:   Fri Feb 10 11:55:46 2017 -0600

    regulator: tps65086: Fix expected switch DT node names
    
    commit 1c47f7c316de38c30b481e1886cc6352c9efdcc1 upstream.
    
    The three load switches are called SWA1, SWB1, and SWB2. The
    node names describing properties for these are expected to be
    the same, but due to a typo they are not. Fix this here.
    
    Fixes: d2a2e729a666 ("regulator: tps65086: Add regulator driver for the TPS65086 PMIC")
    Reported-by: Steven Kipisz <s-kipisz2@ti.com>
    Signed-off-by: Andrew F. Davis <afd@ti.com>
    Tested-by: Steven Kipisz <s-kipisz2@ti.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/regulator/tps65086-regulator.c b/drivers/regulator/tps65086-regulator.c
index caf174f..0e75f57 100644
--- a/drivers/regulator/tps65086-regulator.c
+++ b/drivers/regulator/tps65086-regulator.c
@@ -156,8 +156,8 @@ static struct tps65086_regulator regulators[] = {
 			   VDOA23_VID_MASK, TPS65086_LDOA3CTRL, BIT(0),
 			   tps65086_ldoa23_ranges, 0, 0),
 	TPS65086_SWITCH("SWA1", "swa1", SWA1, TPS65086_SWVTT_EN, BIT(5)),
-	TPS65086_SWITCH("SWB1", "swa2", SWB1, TPS65086_SWVTT_EN, BIT(6)),
-	TPS65086_SWITCH("SWB2", "swa3", SWB2, TPS65086_SWVTT_EN, BIT(7)),
+	TPS65086_SWITCH("SWB1", "swb1", SWB1, TPS65086_SWVTT_EN, BIT(6)),
+	TPS65086_SWITCH("SWB2", "swb2", SWB2, TPS65086_SWVTT_EN, BIT(7)),
 	TPS65086_SWITCH("VTT", "vtt", VTT, TPS65086_SWVTT_EN, BIT(4)),
 };
 
